lib/Cargo.toml

```diff
@@ -31,6 +31,7 @@ openat-ext = "0.2.0"
 openssl = "0.10.33"
 ostree = { features = ["v2021_4"], version = "0.13.0" }
 phf = { features = ["macros"], version = "0.9.0" }
+pin-project = "1.0"
```

Comment:
FYI https://github.com/taiki-e/pin-project-lite.

Reply:
From https://github.com/taiki-e/pin-project-lite#different-no-proc-macro-related-dependencies
I think we have several other proc macros, including `structopt`, `fn-error-context` etc.